doc/users/next_whats_new/2020-03-31-path-size-methods.rst

```diff
@@ -1,12 +1,36 @@
 
-Path extents now computed correctly
------------------------------------
+Functions to compute a Path's size
+----------------------------------
+
+Various functions were added to `~.bezier.BezierSegment` and `~.path.Path` to
+allow computation of the shape, size and area of a `~.path.Path` and its
+composite Bezier curves.
+
+In addition, to the fixes below, `~.bezier.BezierSegment` has gained more
+documentation and usability improvements, including properties that contain its
+dimension, degree, control_points, and more.
+
+Better interface for Path segment iteration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`~.path.Path.iter_bezier` iterates through the `~.bezier.BezierSegment`'s that
+make up the Path. This is much more useful typically than the existing
+`~.path.Path.iter_segments` function, which returns the absolute minimum amount
+of information possible to reconstruct the Path.
+
+Fixed bug that computed a Path's Bbox incorrectly
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Historically, `~.path.Path.get_extents` has always simply returned the Bbox of
-its control points. While this is a correct upper bound for the path's extents,
-it can differ dramatically from the Path's actual extents for non-linear Bezier
-curves.
+a curve's control points, instead of the Bbox of the curve itself. While this is
+a correct upper bound for the path's extents, it can differ dramatically from
+the Path's actual extents for non-linear Bezier curves.
```

Comment:
I think you can arrange this diff better to leave the previously existing two paragraphs unchanged?  Not that it matters too much.

Reply:
Changed up previous PR a bit to make this diff smaller for you ;)